tests/use_cases/list_offers_for_pro_user_test.py

```diff
@@ -0,0 +1,78 @@
+from unittest.mock import MagicMock
+
+from infrastructure.repository.pro_offers.paginated_offer_sql_repository import PaginatedOffersSQLRepository
+from use_cases.list_offers_for_pro_user import ListOffersForProUser, OffersRequestParameters
+
+
+class OffersRequestParametersTest:
+    def should_create_object_with_default_values(self):
+        # When
+        offers_request_parameters = OffersRequestParameters(
+            user_id=12,
+            user_is_admin=True,
+            offerer_id=None,
+            venue_id=None,
+        )
+
+        # Then
+        assert offers_request_parameters.keywords is None
+        assert offers_request_parameters.offerer_id is None
+        assert offers_request_parameters.page == 0
+        assert offers_request_parameters.pagination_limit == 10
+        assert offers_request_parameters.user_id == 12
+        assert offers_request_parameters.user_is_admin is True
+        assert offers_request_parameters.venue_id is None
```

Comment:
```suggestion
        assert offers_request_parameters.keywords is None
        assert offers_request_parameters.page == 0
        assert offers_request_parameters.pagination_limit == 10
```

Vu le nom du test, je m'attendais à ne vois une assertion que sur les valeurs par défaut ?